transport-native-epoll/src/main/java/io/netty/channel/unix/Socket.java

```diff
@@ -428,4 +437,5 @@ private static native DatagramSocketAddress recvFromAddress(
     private static native void setSoLinger(int fd, int soLinger) throws IOException;
     private static native void setTcpDeferAccept(int fd, int deferAccept) throws IOException;
     private static native void setTcpQuickAck(int fd, int quickAck) throws IOException;
+    private static native PeerCredentials peerCredentials(int fd) throws IOException;
```

Comment:
Moved up with the get methods, but did not change the name to getPeerCredentials as I was asked to remove the get prefix a few days ago.